portal/src/graphql/portal/LoginMethodConfigurationScreen.tsx

```diff
@@ -2293,6 +2342,29 @@ function VerificationSettings(props: VerificationSettingsProps) {
         value={verificationConfig.code_expiry_seconds?.toFixed(0) ?? ""}
         onChange={onChangeCodeExpirySeconds}
       />
+      <div className={styles.otpFailedAttemptContainer}>
+        <Toggle
+          label={renderToString(
+            "VerificationConfigurationScreen.otp-failed-attempt.label"
+          )}
+          offText={renderToString(
+            "VerificationConfigurationScreen.otp-failed-attempt.enabled.offText"
+          )}
+          onText={renderToString("Toggle.on")}
+          checked={otpFailedAttemptConfig.enabled ?? false}
+          onChange={onChangeOTPFailedAttemptEnabled}
+        />
+        {otpFailedAttemptConfig.enabled ? (
+          <TextField
```

Comment:
The validation error (min or max validation) is shown in the top bar instead of inline. Please use FormTextField and specify a correct JSON pointer so that the error is shown inline.